subprojects/docs/README.md

```diff
@@ -148,18 +150,25 @@ To run tests for a single sample, let's say from `samples/java/application`:
 ./gradlew :docs:docsTest --tests "org.gradle.docs.samples.DependencyManagementSnippetsTest.java-application*"
 ```
 
-To run tests for a single snippet:
+#### Code snippets
 
-Let's say you want to run the snippet found at `src/snippets/dependencyManagement/customizingResolution-consistentResolution`.
+Snippets live under `src/snippets`.
 
-then you can run the following command line:
+As an example, you can run tests from [`src/snippets/java/toolchain-task/`](src/snippets/java/toolchain-task) using:
+```
+./gradlew :docs:docsTest --tests "*.snippet-java-toolchain-task_*"
+```
 
+This will run both Kotlin and Groovy tests. You can filter the tests for a specific DSL like this:
 ```
-   ./gradlew :docs:docsTest --tests "*.snippet-dependency-management-customizing-resolution-consistent-resolution*"
+./gradlew :docs:docsTest --tests "*.snippet-java-toolchain-task_kotlin_*"
 ```
 
+
 which would run both Groovy and Kotlin tests.
```

Comment:
That line should be removed now

Reply:
For some reason, I was sure you were talking about an extra new line that creeped into the changes 😅 

You are right, I'll fix it.

Reply:
Should be better now!